src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx

```diff
@@ -113,9 +115,15 @@ const individualUnemployabilityContent = (
 );
 
 export default function summaryDescription({ formData }) {
-  const showUnemployableContent =
-    formData['view:unemployable'] &&
-    !formData['view:unemployabilityUploadChoice'];
+  // In production, users should see the unemployable content if they indicate
+  // that they're unemployable. In other environments, they should see the
+  // unemployable content only when no selection has been made for
+  // `view:unemployabilityUploadChoice` (selecting any option for that question
+  // will evaluate to true because all the answers are strings)
+  const showUnemployableContent = environment.isProduction()
+    ? formData['view:unemployable']
+    : !formData['view:unemployabilityUploadChoice'];
```

Comment:
:thinking: If I select no for `view:unemployable`, I won't get to see the `view:unemployabilityUploadChoice`, so that'll be falsey, so this will resolve as true...which means I'll see the 8940 accordion if I don't need to file for it. I think you can leave this bit like it was before with the added comment to clarify the expected behavior.